<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameSenderWindow</class>
 <widget class="QDialog" name="FrameSenderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame Sender</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableSender">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,2,1,1">
     <item>
      <widget class="QPushButton" name="btnEnableAll">
       <property name="text">
        <string>EnAll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDisableAll">
       <property name="text">
        <string>DisAll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearGrid">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReArrange">
       <property name="text">
        <string>Arrange</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoadGrid">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveGrid">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
